src/tools/compiletest/src/runtest.rs

```diff
@@ -2434,7 +2435,33 @@ impl<'test> TestCx<'test> {
         self.add_rmc_dir_to_path(&mut cargo);
         let proc_res = self.compose_and_run_compiler(cargo, None);
         let expected = fs::read_to_string(self.testpaths.file.clone()).unwrap();
-        // Print an error if the verification result does not contains the expected lines.
+        // Print an error if the verification result does not contain the expected lines.
+        if let Some(line) = TestCx::contains_lines(&proc_res.stdout, expected.split('\n').collect())
+        {
+            self.fatal_proc_rec(
+                &format!("test failed: expected output to contain the line: {}", line),
+                &proc_res,
+            );
+        }
```

Comment:
The same code is used for `cargo-rmc` and `expected` so we can refactor into a function.

Reply:
Done!